client/src/components/shared/header/RouteDict.js

```diff
@@ -2,7 +2,7 @@
 // Accessed by headerBackgroundColor() in Header.js
 const routeDict = {
     '/' : 'rgba(0,0,0,0.2)',
-    '/login' : 'white'
+    '/login' : 'white',
```

Comment:
For objects in javascript, we should add trailing commas so that when we add to the object, only the physical line of code we added will pop up in the code diff.

This will more accurately capture how many lines of code a commit affects.